modules/compliance-tests/src-ce3/smithy4s/compliancetests/Compat.scala

```diff
@@ -16,13 +16,20 @@
 
 package smithy4s.compliancetests
 
+import cats.effect.Async
 import cats.effect.Deferred
+import cats.Monad
 import com.comcast.ip4s.Host
 import com.comcast.ip4s.Port
-import cats.effect.Async
+import scala.concurrent.duration.FiniteDuration
 
-private[compliancetests] class CompatEffect[F[_]](implicit val F: Async[F]) {
+private[compliancetests] class CompatEffect[F[_]](implicit
+    val _Async: Async[F],
+    val _Monad: Monad[F]
+) extends CompatUtils[F] {
```

Comment:
I would've just used context bounds tbh

Reply:
I don't think I can't, at least not in the way the code is written right now
because I declare them as `val` here, I can import these implicits in the rest of the code

for example in `ClientHttpComplianceTestCase`, we receive an instance of `ce: CompatEffect` and I can `import ce._` which puts the instances in scope of that we can call things like `request.bodyText` and `thing.pure[F]`

I've gotta say that between my many attempts and rewrite to get the compat layer to work, there is most likely a simpler/better way to achieve the same goal